docs/user-guide/102-policy-syntax.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Policy Syntax"
+tag: "user-guide"
+---
+
+The policy contains the following fields:
+- `capture`: Contains some simple expressions
+to evaluate the current network configuration and store the result at some 
+variables
+- `desiredState`: Contains network state yaml with references to `capture` entries.
+
+__Example:__
+```yaml
+{% include_absolute 'examples/bridge-on-default-gw-dhcp/policy.md' %}
+```
+
+## Capture syntax
+
+The capture is a map of capture entries with an identifier as the key that can 
+be referenced with ```capture.[id]```
+
+The value from the map is a capture entry expression that evaluate the current 
+network state and store the result.
+
+The following is a semi-formal definition of the capture entry expression:
+
+```
+<letter> ::= "A" | "B" | "C" | "D" | "E" | "F" | "G"
```

Comment:
I think this needs to be ordered from high-level down, like man pages are build.

Reply:
For BNF they don't do that the example I have found around like JSON one https://github.com/cierelabs/yaml_spirit/blob/master/doc/specs/json-ebnf.txt start with not composed expressions and continue with composed ones.

Reply:
Well, if you present the info to a machine, like an interpreter, it makes sense. If you present it to a human, he will prefer to see the high level expressions and then dig in on how each is defined.
I think all man-pages are following the same *SYNOPSIS* format, and it seems to fit here as well.

But I will not insist on this, up to you.